app/core/tests/test_models.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+
+from core import models
+
+
+class ModelTests(TestCase):
+
+    def test_recipe_str(self):
+        """Test the recipe string representation"""
+        recipe = models.Recipe.objects.create(
+            name='Pizza',
+            description='Put in the oven'
+        )
```

Comment:
It is possible to create an object without persisting it by using the constructor:
```
models.Recipe(
            name='Pizza',
            description='Put in the oven'
        )
```

That way the test will run faster (more like unit tests) and we won't need to inherit from `TestCase`

Reply:
Thanks for the explanation! now it makes much sense to me :) I don't have access to TK github yet or that doc but when I have I'll take a look for sure.